Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,24 @@
 import React from 'react';
 import './Button.css';
+
+const NumberButton = props => {
+    // const { num } = props;
+    // console.log(props)
+    // const numItems = num.numbers.map((number) =>
+        // <div className="number-item" key={number}>
+        //     {number}
+        //          </div>
+    //             );
+    return (
+        <div>
```

Comment:
this div is extra. probably don't need it

Reply:
ooh ok thanks!